.github/workflows/release.yml

```diff
@@ -21,24 +21,18 @@ on:
   workflow_dispatch:
 jobs:
   release:
-    runs-on: macos-11
+    runs-on: macos-12
     steps:
     - name: Checkout
       uses: actions/checkout@v3
       with:
         token: ${{ secrets.SYNCED_GITHUB_TOKEN_REPO }}
 
     - name: Use Xcode 14.2
-      run: sudo xcode-select -s /Applications/Xcode_14.2.app/Contents/Developer
+      run: sudo xcode-select -s /Applications/Xcode_14.2.app
```

Comment:
Xcode 15 is a requirement for publishing apps since last week, I think you should test there instead of 14.2